net/mwan3/files/lib/mwan3/mwan3.sh

```diff
@@ -1017,7 +1017,7 @@ mwan3_get_iface_hotplug_state() {
 
```

Comment:
You also mentioned in your commit message `don't send track command output in ubus status call`
but i can not find this in this commit?

Reply:
I changed how the files are processed in this loop. `for file in $MWAN3TRACK_STATUS_DIR/${iface}/*; do`  Before, it was taking everything that started with "TRACK_*" and putting it into the ubus status message. This included "TRACK_OUTPUT" which did not need to go there.

Reply:
A ok I understand.
So could you also please do this in another commit so someone else could also follow.